### PR TITLE
feat: Convert github repository policies to be violated by default 

### PR DESCRIPTION
**Pull request by Benjie-legit**
#### What's being changed?

Converting GH repository policies to be violated by default 
#### Is this PR related to an existing issue?

https://github.com/Legit-Labs/legitify/issues/84
#### Check off the following:

- [X] This PR follows the CONTRIBUTION.md guidelines
- [X] I have self-reviewed my changes before submitting the PR

### CODE REVIEW

File: policies/github/repository.rego [hunk 1]
```diff
@@ -11,27 +11,27 @@ import data.common.webhooks as webhookUtils
...
 	now := time.now_ns()
 	diff := time.diff(now, ns)
 	monthsIndex := 1
 	inactivityMonthsThreshold := 3
-	diff[monthsIndex] >= inactivityMonthsThreshold
+	diff[monthsIndex] < inactivityMonthsThreshold
 }
 
-repository_not_maintained {
+repository_not_maintained = false {
```
Comment (noamd-legit) [inline policies/github/repository.rego:27]: merge with the above

File: policies/github/repository.rego [hunk 2]
```diff
@@ -105,11 +105,11 @@ repository_webhook_doesnt_require_ssl[violated] = true {
...
 #   requiredScopes: [read:org]
 #   threat: Forked repositories cause more code and secret sprawl in the organization as forks are independent copies of the repository and need to be tracked separately, making it more difficult to keep track of sensitive assets and contain potential incidents.
-default forking_allowed_for_repository = false
+default forking_allowed_for_repository = true
 
-forking_allowed_for_repository {
-	input.repository.is_private == true
-	input.repository.allow_forking == true
+forking_allowed_for_repository = false {
+	input.repository.is_private == false
```
Comment (noamd-legit) [inline policies/github/repository.rego:111]: this policy applies for private repositories
```suggestion
	input.repository.is_private == true
```

File: policies/github/repository.rego [hunk 3]
```diff
@@ -146,14 +146,11 @@ missing_default_branch_protection {
...
+default missing_default_branch_protection_deletion = true
 
-missing_default_branch_protection_deletion {
-	missing_default_branch_protection
-}
-
-missing_default_branch_protection_deletion {
-	input.repository.default_branch.branch_protection_rule.allows_deletions == true
+missing_default_branch_protection_deletion = false {
+	not missing_default_branch_protection
```
Comment (noamd-legit) [inline policies/github/repository.rego:152]: not needed

Comment (noamd-legit) [reply policies/github/repository.rego:152]: delete in other policies as well

File: policies/github/repository.rego (hunk 1 above)
Comment (Benjie-legit) [reply policies/github/repository.rego:27]: done

File: policies/github/repository.rego (hunk 3 above)
Comment (Benjie-legit) [reply policies/github/repository.rego:152]: Done

**Review by noamd-legit — APPROVED** [review]

**Pull request merged**